### ISSUE TITLE
SVDKL training by gradient accumulation trick

### ISSUE THREAD

**Issue by Kangfei** [cornellius-gp/gpytorch#1397]
I'm using a stochastic variational DKL for batch training a regression model.  As my specific feature extractor is hard to support batched training data. I used Pytorch gradient accumulation trick, e.g., https://gist.github.com/thomwolf/ac7a7da6b1888c2eeac8ac8b9b05d3d3. I'm wondering whether the variational inference can support that training mode, as it involves kernel interpolation.

**Comment by gpleiss** [comment]
It does support that trick. That trick is simply an autograd trick (not neural network specific).
